packages/react-components/src/components/MessageThread.tsx

```diff
@@ -998,17 +1009,24 @@ export const MessageThread = (props: MessageThreadProps): JSX.Element => {
 
   return (
     <Ref innerRef={chatThreadRef}>
-      <Stack className={mergeStyles(messageThreadContainerStyle, styles?.root)} grow>
-        <Ref innerRef={chatScrollDivRef}>{chatBody}</Ref>
-        {existsNewChatMessage && !disableJumpToNewMessageButton && (
-          <div className={mergeStyles(newMessageButtonContainerStyle, styles?.newMessageButtonContainer)}>
-            {onRenderJumpToNewMessageButton ? (
-              onRenderJumpToNewMessageButton({ text: strings.newMessagesIndicator, onClick: scrollToBottom })
-            ) : (
-              <DefaultJumpToNewMessageButton text={strings.newMessagesIndicator} onClick={scrollToBottom} />
-            )}
-          </div>
-        )}
+      <Stack grow>
+        {/* @conditional-compile-remove(file-sharing) */}
+        <FileDownloadErrorBar
```

Comment:
This is not working. If it was, you would get an error for unused import of `FileDownloadErrorBar` :P
To remove this you would need to change it to 
```tsx
{
  /* @conditional-compile-remove(file-sharing) */
  <FileDownloadErrorBar ... </FileDownloadErrorBar>
}
```

Reply:
This is a fairly big gotcha. @prabhjot-msft can you also update https://github.com/Azure/communication-ui-library/blob/main/packages/acs-ui-common/src/conditional-compilation-sample/index.tsx#L155 with how *not* to remove JSX elements

(Separate PR if you do that) 

Thanks.